includes/transformer/class-post.php

```diff
@@ -894,11 +894,20 @@ protected function get_post_content_template() {
 		$template = $content ?: ACTIVITYPUB_CUSTOM_POST_CONTENT; // phpcs:ignore Universal.Operators.DisallowShortTernary.Found
 
 		$post_format_setting = \get_option( 'activitypub_object_type', ACTIVITYPUB_DEFAULT_OBJECT_TYPE );
+		$type                = $this->get_type();
 
 		if ( 'wordpress-post-format' === $post_format_setting ) {
 			$template = '';
 
-			if ( 'Note' === $this->get_type() ) {
+			/*
+			 * If the post is a note, not a reply, and does not have mentions
+			 * force the inclusion of the post title.
+			 */
+			if (
+				'Note' === $type
+				&& empty( $this->get_in_reply_to() )
+				&& empty( $this->get_mentions() )
+			) {
```

Comment:
Shouldn't it be either or?

```suggestion
				'Note' === $type &&
				( empty( $this->get_in_reply_to() ) || 
				empty( $this->get_mentions() ) )
			) {
```

Reply:
Oh no, it checks for empty! You are right!!! Forget my comment 🫣